airflow_munchkin/client_parser/cloud_client_parser.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+import inspect
+from typing import Callable, List, Dict, Optional
+
+from airflow_munchkin.client_parser import docstring_parser
+from airflow_munchkin.client_parser.docstring_parser.bricks import FieldBrick, TypeBrick
+from airflow_munchkin.client_parser.infos import ParameterInfo, ActionInfo, ClientInfo
+from airflow_munchkin.utils import load_class
+
+
+def parse_method(name: str, method: Callable) -> ActionInfo:
+    docstring = method.__doc__
+    assert docstring is not None
+
+    doc_sections = docstring_parser.parse_docstring(docstring)
+    desc: Optional[List[str]] = None
+    args: Dict[str, ParameterInfo] = {}
+    return_kind: Optional[TypeBrick] = None
+    return_desc: Optional[List[str]] = None
+    for i, section in enumerate(doc_sections):
+        if i == 0 and section.kind == "Text":
+            assert isinstance(section.body, list)
+            desc = section.body  # type: ignore
+        elif section.kind.lower() in ("args", "arguments", "parameters"):
+            for arg in section.body:
+                if not isinstance(arg, FieldBrick):
+                    raise Exception
+                args[arg.name] = ParameterInfo(
+                    name=arg.name, kind=arg.type_brick, desc=arg.desc[0].body
+                )
+        elif section.kind.lower() in ("return", "returns"):
+            if not isinstance(section.body, list) or len(section.body) != 1:
+                raise Exception
```

Comment:
It will change to assertions.  Then the message will be clearer. This state will never happen in the normal code, because every this type of section always contains a list with single element.